docs/source/installation-guide-hard.md

```diff
@@ -143,7 +143,7 @@ After=syslog.target network.target
 
 [Service]
 User=root
-Environment="PATH=/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/opt/jupyterhub/bin"
+Environment="PATH=/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/usr/bin/sudo:/opt/jupyterhub/bin"
```

Comment:
/usr/bin/sudo is a directory containing executables on your system?

Reply:
Actually I faced peculiar issue.when I was running jupyter hub service from root user.it was not creating session for users.so I provided /usr/bin/sudo to run as "sudo jupyterhub" and it works in my ubuntu environment 18.04